src/main.go

```diff
@@ -33,12 +33,150 @@ func getOlPath(ctx *cli.Context) (string, error) {
 	return filepath.Abs(olPath)
 }
 
+func getOlWorkerProc(ctx *cli.Context) (int, error) {
+	// Check all running ol procs, substracting "this" one.
+	// Return:
+	//	pid:
+	// 		Return the process id if one proc is working.
+	//		Return 0 if no proc is working.
+	//		Return < 0 as error code.
+	//	err:
+	//		Error message of all kind.
+
+	// Assumption:
+	//	1. At most one ol worker is running in the system.
+	//  2. This process is the only other ol admin process that could be running.
+	// TODO: Assumption 2 is too strong as there could be many `ol status` running.
+
+	// Get the running pid whose program name is `ol` (at least one - this one)
+	out, err := exec.Command("ps", "-C", "ol", "-o", "pid=").Output()
```

Comment:
Problem here:
1. We assume the executable will be named `ol`. This hard-code value is probably not a good idea, and we shall specify the path to the `ol` command.
2. There could be multiple `ol` running at the same time (multiple admins doing status check, etc). To discern the cases we need more robust mechanism to do this